src/btree/bt_sync.c

```diff
@@ -221,8 +227,46 @@ __sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 				leaf_bytes += page->memory_footprint;
 				++leaf_pages;
 			}
+			++page_counter;
+
 			WT_ERR(__wt_reconcile(
 			    session, walk, NULL, WT_CHECKPOINTING, NULL));
+
+			if (timer && page_counter > 5000) {
+				__wt_epoch(session, &cur_time);
+				time_diff = WT_TIMEDIFF_SEC(cur_time,
+						conn->ckpt_tree_verb_time);
+
+				if (time_diff > 20) {
```

Comment:
use a macro instead of hardcoding a number

Reply:
Since the number is specific to this specific logic, not created a macro for it.